classes/tests/test_events.py

```diff
@@ -62,27 +81,31 @@ def test_scooter_arrival(self):
         self.assertEqual(self.world.time, self.departure_time + self.travel_time)
 
         # test if arrival cluster contains the arrived scooter
-        self.assertTrue(arrival_cluster.scooters.__contains__(scooter))
+        self.assertIn(scooter, arrival_cluster.scooters)
 
         # test if battery has decreased
         self.assertLess(scooter.battery, scooter_battery)
 
     def test_vehicle_arrival(self):
-        random_cluster_in_state = self.world.state.get_random_cluster(
-            exclude=self.world.state.current_cluster
+        # Choose a random cluster for the vehicle to be in
+        arrival_cluster = self.large_world.state.get_random_cluster(
+            exclude=self.vehicle_large_world.current_location
         )
+        self.vehicle_large_world.current_location = arrival_cluster
```

Comment:
Why do you set vehicle location to arrival cluster? This is what we want to test with perform